### ISSUE TITLE
No qubes windows show

### ISSUE THREAD

**Issue by deeplow** [raffaeleflorio/qubes-dwm#2]
See the problem here: https://github.com/QubesOS/qubes-issues/issues/5966

**Comment by deeplow** [comment]
solution was to run ` qvm-start-gui --all --watch` as pointed out in thread

**Comment by raffaeleflorio** [comment]
Great! Precisely when should be run it?

**Comment by deeplow** [comment]
Thanks for replying @raffaeleflorio. I ran it manuall right after it started following [this suggestion](https://github.com/QubesOS/qubes-issues/issues/5966#issuecomment-773300912). But I feel like this should be run automatically on startup. I just don't know where it would fit the best.

**Comment by jsbmg** [comment]
You could launch dwm with a script that launches other stuff too, instead of launching dwm directly from the desktop entry.

File "/usr/local/bin/launchdwm":
```
#! /bin/sh

# launch qubes autostart programs
# see https://gist.github.com/SietsevanderMolen/7b4cc32ce7b4884513b0a639540e454f

autostart_etc=${XDG_CONFIG_DIRS-/etc/xdg}/autostart
autostart_home=${XDG_CONFIG_HOME-~/.config}/autostart

shopt -s nullglob
for i in $autostart_etc/*.desktop $autostart_home/*.desktop; do
    if ! grep -q "OnlyShowIn=" "$i"; then
        $(grep "Exec=" "$i" | sed 's/Exec=//') &
    fi
done &
  
# start dwm
while xsetroot -name "`date` `uptime | sed 's/.*,//'`"
do
    sleep 1
done &
exec /usr/local/bin/dwm
```

In dwm.desktop:
```
"..."
Exec=/usr/local/bin/launchdwm
"..."
```

**Comment by raffaeleflorio** [comment]
@jsbmg thanks for the comment! Furthermore it seems that dex-autostart is not needed in this way...
What do you think about a pull request?

**Comment by jsbmg** [comment]
@raffaeleflorio just created one. Let me know if you need me to make any changes!

**Comment by raffaeleflorio** [comment]
Issue resolved by #4
